src/element.py

```diff
@@ -0,0 +1,49 @@
+from periodic_table import PERIODIC_TABLE
+
+
+class Element:
+
+
+    element_dic = {}
```

Comment:
We dont need to define these here if we set them in the init method so we can remove these